picoscope/picobase.py

```diff
@@ -247,10 +247,12 @@ def setChannel(self, channel='A', coupling="AC", VRange=2.0,
         if not isinstance(BWLimited, int):
             BWLimited = self.BW_LIMITS[BWLimited]
 
-        if BWLimited == 2:
```

Comment:
we should just raise an error if the BWLimit is not in the the values of the dictionary above. That can be left as an issue for an other time.

Reply:
or use something like a default dict, to get BWLimit 0.

Reply:
don't  worry about addressing this now, it is more for future reference.

Reply:
Should we refactor it so the values of that dict come from the device classes rather then the base class. As it seems the enum values (dict) have different meanings device to device? Reading the documentation for the 4444 it also appears that vary depending on what probes are attached. I.e. I get an error if I use the 20kHz setting with any of the probes I have for the 4444, but I assume there must be a probe that does support it, or why would it be in the API.